funnel/templates/index.html.jinja2

```diff
@@ -68,7 +68,7 @@
       <div class="mui-container">
         <div class="grid">
           <div class="grid__col-12">
-            <a class="card card--spotlight clickable-card" href="https://hasgeek.com/PrivacyMode" aria-label="PrivacyMode" data-cy="spotlight-project" data-ga="View PrivacyMode">
+            <a class="card card--spotlight clickable-card" href="https://hasgeek.com/PrivacyMode" aria-label="PrivacyMode" data-ga="View PrivacyMode">
```

Comment:
The brand has a space in it, so this should be "Privacy Mode" in the ARIA label and GA data.